client/aws/aws_test.go

```diff
@@ -19,32 +20,41 @@ func Test_Extraction(t *testing.T) {
 		t.Fail()
 	}
 
-	getPriceStart := time.Now()
-	priceList, err := extractPrice(rawData)
+	getOfferStart := time.Now()
+	offerList, err := extractOffer(rawData)
 	if err != nil {
 		t.FailNow()
 	}
-	getPriceEnd := time.Now()
-	fmt.Printf("get pricing info: [entry cnt]: %v, [time duration]: %v\n", len(priceList), getPriceEnd.Sub(getPriceStart))
+	getOfferEnd := time.Now()
+	fmt.Printf("get pricing info: [entry cnt]: %v, [time duration]: %v\n", len(offerList), getOfferEnd.Sub(getOfferStart))
 
-	getInstanceStart := time.Now()
-	instanceList, err := extractInstance(rawData)
+	fillInstanceStart := time.Now()
+	byteProducts, err := json.Marshal(rawData.Product)
 	if err != nil {
 		t.FailNow()
```

Comment:
Follow other tests at Bytebase, use "require.NoError" for example.

https://github.com/bytebase/bytebase/blob/main/tests/schema_update_test.go